joomla/joomla-cms
# [5.0] Replace hard coded img element by JLayout joomla.html.image and DRY

Pull Request for no Issue

### Summary of Changes

This PR will change the hard coded `<img src=""...` inside the default view of mod_banners by a call to JLayout joomla.html.image and keeps it DRY.

### Testing Instructions

- Joomla 5 with testing sample data. (adjust mod_banners config and unset the chosen client, otherwise you don't see images... https://github.com/joomla/joomla-cms/issues/40616)
- Go to frontend index.php/banners 

### Actual result BEFORE applying this Pull Request

You will see a banner image on the page. 

### Expected result AFTER applying this Pull Request

You will see a banner image on the page.  The same as BEFORE. The HTML output is also the same as BEFORE. 
The only difference is that it is rendered by JLayout which allows users to create an override. 

### Link to documentations
Please select:
